cmd/minikube/cmd/config/unset.go

```diff
@@ -25,8 +25,9 @@ import (
 
 var configUnsetCmd = &cobra.Command{
 	Use:   "unset PROPERTY_NAME",
-	Short: "unsets an individual value in a minishift config file",
-	Long:  "unsets PROPERTY_NAME from the minishift config file.  Can be overwritten by flags or environmental variables",
+	Short: "Clears the value of a configuration property in the Minishift configuration file.",
+	Long:  "Clears the value of a configuration property in the Minishift configuration file. The value can be overwritten at runtime by flags or environmental variables",
```

Comment:
And either way I need to document expected behavior, if there's a bug in this function we should make a note of it somewhere but the intention/expected behavior should be documented here.